---
title: Research
description:   
background: /assets/img/1_14.jpg
permalink: /about/
---

## Molecular dialogue between plant and fungal pathogen

Plants have evolved sophisticated mechanisms to recognize non-self molecules, allowing them to deploy effective immune reactions against a myriad of pathogens. Plant PRRs (Pattern Recognition Receptor) and NLRs (Nucleotide-Binding Leucine Rich Repeat) sense pathogen-derived molecules PAMPs (Pathogen-associated molecular pattern) and effectors, respectively, to activate many signaling events to coordinate antimicrobial defenses (Li et al., Cell Host Microbe 2014, 2016, 2020).

Fungal pathogens cause severe loss to crop productions. How do plants and fungal pathogens regulate their molecular processes during the interaction? What kind of molecules are involved in the communication between plant and fungal pathogen, and how are they transported between each other? We will use _Arabidopsis-Colletotrichum higginsianum_ and _Rice-Magnaporthe oryzae_ system to investigate the molecular dialogue between plants and microbes. We also will study effectors-mediated immunity at the pan-genome level to understand the co-evolution between plants and their pathogens.
![colleto](/assets/img/colleto.jpg){: .rounded .float-left}

---

## Plant disease resistance genes 

The era of big data allows us to study plant-microbe interaction systematically from genome to phenome and back. In plant pathology, precise, high-throughput, and reproducible phenotyping is still a big obstacle, particularly in the case of disease caused by fungal pathogens. We are interested in developing plant phenomics approaches for automatic, non-destructive and high-throughput phenotyping.


The evolutionary relationship between plants and microbes has dramatically affected their genomes and result in highly diverse populations. Immune genes are the most variable genes in plant genomes, and likewise the virulence genes are the most variable genes in the pathogen genomes. We are interested in using the wild crop relatives and/or progenitor, and field samples to identify resistance genes from host and (a)virulence genes from pathogen, and using genomic method to understand how microbial populations drive the evolution of plant immune genes. In the end, We want to use this newly generated resources to engineer disease resistance in crops.
![pheno](/assets/img/pheno.png){: .rounded .float-left}

---

## Plant endophytic fungal communities

Each plant species harbors a core microbial community, which plays a critical function in plant growth and resistance. Some recent studies about human and plant microbiome show that metabolites from both host and microbe can affect counterpart. For example, nonribosomal peptides and polyketides produced by microbes may regulate microbe-microbe or microbe-plant-microbe interactions in plant disease suppression. Focusing on plant-associated fungal community, we will isolate and culture endophytic fungal strains to study the higher-order interaction between different kingdoms. Specifically, We are interested in functional researches by investigating natural endophytic fungal isolates in plant disease resistance. Our goal is to improve plant resistance through manipulating the natural fungal community by genetic engineering tools.
![tree](/assets/img/tree.jpg){: .rounded .float-left}


